packages/client/src/components/Button/ProductImage/ProductImage.js

```diff
@@ -0,0 +1,33 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import './ProductImage.styles.css';
+
+function ProductImage({ topSide, middleSide, bottomSide, bigImage }) {
+  return (
+    <div className="wrapper">
```

Comment:
This might in the future clash with another component if somebody also decides to use "`wrapper`".
So maybe "`product-image-wrapper`"?

Reply:
Yes I agree, I will change it 👍